common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -150,20 +214,39 @@ def add_cohort_with_student(cohort_name, content_group, student):
 
     def view_cohorted_content_as_different_users(self):
         """
-        View content as staff, student in Cohort A, student in Cohort B, and student in Default Cohort.
+        View content as staff, student in Cohort A, student in Cohort B, Verified Student, Audit student,
+        and student in Default Cohort.
         """
         courseware_page = CoursewarePage(self.browser, self.course_id)
 
-        def login_and_verify_visible_problems(username, email, expected_problems):
+        def login_and_verify_visible_problems(username, email, expected_problems, track=None):
             LmsAutoAuthPage(
                 self.browser, username=username, email=email, course_id=self.course_id
             ).visit()
+            if track is not None:
+                set_user_track(track)
             courseware_page.visit()
             verify_expected_problem_visibility(self, courseware_page, expected_problems)
 
+        def set_user_track(track=None):
+            payment_and_verification_flow = PaymentAndVerificationFlow(self.browser, self.course_id)
```

Comment:
If this same code exists someplace else, please create a helper method that multiple tests can reference.